assets/dev/js/editor/components/browser-import/container-factory.js

```diff
@@ -1,17 +1,20 @@
 import ElementModel from '../../elements/models/element';
 
-export default class ContainerFactory {
+/**
+ * @typedef {import('../../container/container')} Container
+ */
+ export default class ContainerFactory {
 	/**
 	 * Create an element container.
 	 *
-	 * @param element
-	 * @returns {Container}
+	 * @param {*} element
+	 * @return {Container} element container
```

Comment:
Why do we need a rule for return value description? It's annoying and redundant in my opinion, since the method name itself is descriptive enough

Reply:
Its a default setting in the `@wordpress/eslint-plugin` module's `recommended` settings.
History (current code) shows that the method name is not always descriptive enough.
BUT we don't need to JSDoc every function - only APIs & complex/"important" functions. APIs need high documentation standards, complex functions usually need high documentation standards as well.